data/json/items/comestibles/alcohol.json

```diff
@@ -1425,6 +1425,33 @@
     "fun": 25,
     "vitamins": [ [ "vitA", 2 ], [ "vitC", 23 ], [ "calcium", 1 ], [ "iron", 2 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "drink_bloody_mary_blood",
+    "name": { "str_sp": "'exalted' bloody mary cocktail" },
+    "weight": "258 g",
+    "color": "red",
+    "addiction_type": "alcohol",
+    "use_action": [ "ALCOHOL" ],
+    "stim": -12,
+    "container": "bottle_glass",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 25,
+    "healthy": -1,
+    "addiction_potential": 4,
+    "calories": 240,
+    "description": "This is definitely not colored with tomato.",
```

Comment:
```suggestion
    "description": "A sickly red bloody Mary cocktail. Certainly not made the standard way.",
```

Reply:
I want to keep the description style consistent with `brownie_weed`.